includes/agentic-commerce/class-wc-stripe-agentic-commerce-tax-calculator.php

```
@@ -0,0 +1,176 @@
+	 * @since 10.5.0
+	 * @param string                 $line_item_id The line item ID.
+	 * @param string                 $product_id   The product ID.
+	 * @param WC_Stripe_API_Address  $address      The tax address.
+	 * @return array The line item tax response.
+	 */
+	private function calculate_line_item_taxes(
+		string $line_item_id,
+		string $product_id,
+		WC_Stripe_API_Address $address
+	): array {
+		if ( '' === $product_id ) {
```

PHPMD (2.15.0)

[error] The variable $product_id is not named in camelCase. (undefined)

(CamelCaseVariableName)